lib/uiautomator2.js

```diff
@@ -119,9 +120,15 @@ class UiAutomator2Server {
       this.startSessionUsingAdbkit();
     }
 
-    logger.info('Waiting for UiAutomator2 to be online...');
+    let retries = SERVER_LAUNCH_RETRIES;
+    if (caps.uiautomator2ServerLaunchTimeout) {
+      caps.uiautomator2ServerLaunchTimeout = parseInt(caps.uiautomator2ServerLaunchTimeout, 10);
+      retries = Math.round(caps.uiautomator2ServerLaunchTimeout / 1000);
+    }
+
+    logger.info(`Waiting up to ${retries * 1000}ms for UiAutomator2 to be online...`);
     // wait 20s for UiAutomator2 to be online
-    await retryInterval(20, 1000, async () => {
+    await retryInterval(retries, 1000, async () => {
```

Comment:
`retryInterval` is pretty smart. you can actually do this:

```js
await retryInterval(retries, 1000, this.jwproxy.command, '/status', 'GET');
```

to make it a one-liner! (assuming binding stuff is a-ok)